### PR TITLE
increase contrast of links

### PR DESCRIPTION
**Pull request by maelle**
cf #445

### CODE REVIEW

File: themes/ropensci/static/css/style-new.css [hunk 1]
```diff
@@ -52,106 +52,83 @@ q:before, q:after {
...
 }.col-1{
   box-sizing: border-box;
-  -webkit-box-flex: 0;
-          flex: 0 0 auto;
+  flex: 0 0 auto;
   padding-left: .25%;
   padding-right: .25%;
 }.col-2{
   box-sizing: border-box;
-  -webkit-box-flex: 0;
```
Comment (maelle) [inline themes/ropensci/static/css/style-new.css]: not sure why this was changed, maybe a css/npm update?

**Comment by maelle** [comment]
More should be done, cf http://wave.webaim.org/report#/https://deploy-preview-448--ropensci.netlify.com/blog (click on contrast, although another time we should have a look at other alerts!)

* color of "by", date.

 * the header, as pretty as it is, has low contrast too :crying_cat_face: 

* the tags and tech note table header

Any suggestions of new colors for these items?

Once we make decisions for ropensci.org I'll apply them to the dev_guide, and our [pkgdown template](https://ropensci.github.io/rotemplate/).

**Comment by jeroen** [comment]
This is a big improvement. Lets merge and discuss additional tweaks in a new issue.

**Comment by maelle** [comment]
A new PR you mean? #445 is general enough I think?

File: themes/ropensci/static/css/style-new.css (hunk 1 above)
Comment (karthik) [reply themes/ropensci/static/css/style-new.css]: It's possible but Im not sure.

**Comment by karthik** [comment]
Merged before I could review.

**Pull request merged**